### PR TITLE
Update: licensing prompt dialog wording

### PR DESCRIPTION
**Pull request by andrii-lysenko**
#### Changes proposed in this Pull Request

Some wording should be changed in the licensing prompt dialogue window when you have detached licenses.
- Description can be found in the task: 1164141197617539-as-1201710570188036/f

#### Testing instructions
Scenario 1:
Shows `Activate {productName}` in dialog window header and body: `Find the license key in your purchase confirmation email to activate your new Jetpack features.`
- Create a new JN site and connect to Jetpack.
- Navigate to https://cloud.jetpack.com/pricing and purchase 1 plan.
- After checkout, don't attach it to a site.
- Navigate to [Calypso](https://calypso.live?image=registry.a8c.com/calypso/app:commit-444d09929b064b33ad872a52fabf2296443da499) and set relative path to `/jetpack/connect/plans/${your-JN-site}`
##### Before
Screenshot 2022-01-26 at 14 48 53

##### After
Screenshot 2022-01-26 at 16 00 10

Scenario 2 (after Scenario 1 is done):
Shows `Activate your new Jetpack features` in dialog window header and body: `Find the license key in your purchase confirmation email to activate your new Jetpack features.`
- Navigate to https://cloud.jetpack.com/pricing and purchase another plan.
- After checkout, don't attach it to a site.
- Navigate to [Calypso](https://calypso.live?image=registry.a8c.com/calypso/app:commit-444d09929b064b33ad872a52fabf2296443da499) and set relative path to `/jetpack/connect/plans/${your-JN-site}`
##### Before
Screenshot 2022-01-26 at 14 50 12

##### After
Screenshot 2022-01-26 at 15 58 16

### CODE REVIEW

File: client/components/jetpack/licensing-prompt-dialog/index.tsx [hunk 1]
```diff
@@ -49,15 +49,15 @@ function LicensingPromptDialog( { siteId }: Props ) {
 		if ( hasOneDetachedLicense ) {
 			return detachedUserLicense?.product
 				? preventWidows(
-						translate( 'Your %(productName)s is pending activation', {
+						translate( 'Activate %(productName)s', {
```
Comment (a8ci18n) [inline client/components/jetpack/licensing-prompt-dialog/index.tsx:52]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Activate your %(productName)s plan' )` *ES Score: 6*

Comment (kallehauge) [reply client/components/jetpack/licensing-prompt-dialog/index.tsx:52]: We could be activating either a plan or a product, so the suggested translation is too specific and do not work for us here.

**Review by kallehauge — COMMENTED** [review]
I went through the Google Document again and it looks like we missed the "body" change that appears in the next page 🙂

**Comment by andrii-lysenko** [comment]
> I went through the Google Document again and it looks like we missed the "body" change that appears in the next page 🙂

Thank you. The body is updated as well now.

**Review by manzoorwanijk — APPROVED** [review]
Works as expected ✔️

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7120040

Thank you @andrii-lysenko for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7120040).

**Pull request merged**